app/controllers/admins_controller.rb

```diff
@@ -64,9 +60,8 @@ def verify
     if voter.save
       send_email = params[:send_email] == "true"
       if send_email
-        # Will need to be replaced with deliver_now
         begin
-          UserMailer.voter_verified(voter, event_year).deliver
+          UserMailer.voter_verified(voter, event_year).deliver_now
```

Comment:
?

$ rails -v
/usr/local/bin/rails:23:in `load': cannot load such file -- /usr/share/rubygems-integration/all/gems/railties-4.1.8/bin/rails (LoadError)
	from /usr/local/bin/rails:23:in `<main>'
owilliams@grants:~$ which rails
/usr/local/bin/rails
owilliams@grants:~$ /usr/bin/rails -v
Rails 4.1.8